spec/javascripts/Dashboard/components/NewAccountsWidget.spec.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import React from 'react'
+
+import { NewAccountsWidget, Props } from 'Dashboard/components/NewAccountsWidget'
+import { mount } from 'enzyme'
+
+const defaultProps: Props = {
+  chartData: {
+    values: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    },
+    complete: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    },
+    incomplete: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    },
+    previous: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    }
+  },
+  newAccountsTotal: 10,
+  hasHistory: true,
+  links: {
+    previousRangeAdminBuyersAccount: {
+      url: '/buyers/accounts?search%5Bcreated_within%5D%5B%5D=2021-11-03&search%5Bcreated_within%5D%5B%5D=2021-12-03',
+      value: '0'
+    },
+    currentRangeAdminBuyersAccount: {
+      url: '/buyers/accounts?search%5Bcreated_within%5D%5B%5D=2021-12-03&search%5Bcreated_within%5D%5B%5D=2022-01-03'
+    },
+    lastDayInRangeAdminBuyersAccount: {
+      url: '/buyers/accounts?search%5Bcreated_within%5D%5B%5D=2022-01-03&search%5Bcreated_within%5D%5B%5D=2022-01-03',
+      value: '0'
+    }
+  },
+  percentualChange: 50
+}
+
+const mountWrapper = (props: Props) => mount(<NewAccountsWidget {...{ ...defaultProps, ...props }}/>)
+
+it('should render', () => {
+  const wrapper = mountWrapper(defaultProps)
+
+  expect(wrapper.exists()).toBe(true)
+})
+
+describe('when there is historical data', () => {
+  describe('and the percentual change is positive', () => {
+    it('should render past data', () => {
+      const wrapper = mountWrapper(defaultProps)
+
+      expect(wrapper.find('.DashboardWidget-link.u-plus').exists()).toBe(true)
+    })
+  })
+
+  describe('and the percentual change is negative or 0', () => {
+    it('should render past data', () => {
+      let props: Props = { ...defaultProps, percentualChange: 0 }
```

Comment:
Since these props won't change, I'd suggest using `const` instead of `let`. Applies to all `let props`.

```suggestion
      const props: Props = { ...defaultProps, percentualChange: 0 }
```

Reply:
Updated in bf00c23.